pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/SchemasResourceBase.java

```diff
@@ -97,7 +98,7 @@ public void getSchema(boolean authoritative, AsyncResponse response) {
     }
 
     public CompletableFuture<SchemaAndMetadata> getSchemaAsync(boolean authoritative) {
-        return validateDestinationAndAdminOperationAsync(authoritative)
+        return validateOwnershipAndOperationAsync(authoritative, TopicOperation.CONSUME)
```

Comment:
For both PRODUCE and CONSUME, it can get schema, not only the consumer.

Reply:
Could we introduce a new operation `GET_SCHEMA`? use `canProduce` and `canConsume` to verify it?

The combination mode may bring more flexibility.

Reply:
Ah, now we can use `GET_METADATA` instead.

Reply:
> For both PRODUCE and CONSUME, it can get schema, not only the consumer.

fixed